### PR TITLE
Add getting started with organizations

### PR DESCRIPTION
**Pull request by go12**
this will be an interesting experiment ;)
~1500 words and 17 images!
hidden flag is set to true

### CODE REVIEW

File: _articles/organizations/getting-started-organizations.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+{% image organizations/orgstart/6-user-2-organization-vault-view.png %}
+**User 2 Organization Vault view**
+## 7. Sharing within an Organization
+Sharing items in an Organization is simply a matter of creating them within the Organization Vault, or creating them in your Personal Vault and then sharing them with the Organization.
+
+We’ll use the example of creating a Secure Note, but the same process applies to a Login or any other item in the vault.
+
+{% image organizations/orgstart/7-add-Item-in-organization.png %}
+**Adding an Item to an Organization)
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]: missing closing `**`

```suggestion
**Adding an Item to an Organization**
```

File: _articles/organizations/getting-started-organizations.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  * [1. Create an account for User 1](#1-create-an-account-for-user-1)
+  * [2. Verify Your Email](#2-verify-your-email)
+  * [3. Set up the Second User Account](#3-set-up-the-second-user-account)
+- [Setup and Share with Organizations](#setup-and-share-with-organizations)
+  * [4. Create a new Organization](#4-create-a-new-organization)
+  * [5. Add the Second User to your Organization](#5-add-the-second-user-to-your-organization)
+  * [6. Accepting the Invitation](#6-accepting-the-invitation)
+  * [7. Sharing within an Organization](#7-sharing-within-an-organization)
+- [Continuing with Bitwarden](#continuing-with-bitwarden)
+# Introduction
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]:
```suggestion

# Introduction
```

New line

File: _articles/organizations/getting-started-organizations.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+Using a password manager makes it easy to store and access unique and secure passwords across all of your devices. This keeps your online accounts safer than repeating passwords or leaving them exposed in unencrypted formats like spreadsheets, documents, or sticky notes!
+
+To get even more out of your password manager, you can collaborate and share with others. At work, you may choose to share common logins to office wifi networks and online services, or a shared company credit card, or a set of secure notes. Personally you may choose to share a few passwords with a significant other.
+
+Bitwarden enables all types of password and secure sharing through Organizations. 
+
+If you would like to try it out, Bitwarden has a free account for individuals, and the ability to set up a free Organization for up to two users and two Collections of secure information.
+
+This tutorial walks you through the setup for two basic accounts and sharing in an Organization.
+## What are organizations?
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]: I prefer to keep new lines around headings.

```suggestion

## What are organizations?

```

File: _articles/organizations/getting-started-organizations.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+A secure note **User 1 Note for Sharing** that is not part of Collection and only within the **User 1** vault
+
+However, that second note can easily be shared with an Organization by clicking the drop down menu next to the item.
+
+{% image organizations/orgstart/7-sharing-items-with-organizations.png %}
+**Sharing Items with Organizations**
+# Continuing with Bitwarden
+Now that you have mastered the basics of Organizations feel free to explore all of the possibilities sharing logins and more!
+
+If you’d like to dig deeper into Bitwarden please visit [www.bitwarden.com](www.bitwarden.com).
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]:
```suggestion
If you’d like to dig deeper into Bitwarden please visit [bitwarden.com](https://bitwarden.com).
```

File: _articles/organizations/getting-started-organizations.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+
+**Are Organizations Different than a Premium Membership?**
+Yes. Organizations are for sharing items across multiple users
+Premium membership gives your individual user account additional premium features, including
+1GB encrypted file storage
+Two-step login with YubiKey, FIDO U2F, & Duo
+Password hygiene & vault health reports
+TOTP authenticator key storage & code gen.
+Priority customer support
+# Set up the First and Second User Accounts
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]:
```suggestion

# Set up the First and Second User Accounts

```

File: _articles/organizations/getting-started-organizations.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+For more, please see [What are organizations?](https://help.bitwarden.com/article/what-is-an-organization/)
+
+**Are Organizations Different than a Premium Membership?**
+Yes. Organizations are for sharing items across multiple users
+Premium membership gives your individual user account additional premium features, including
+1GB encrypted file storage
+Two-step login with YubiKey, FIDO U2F, & Duo
+Password hygiene & vault health reports
+TOTP authenticator key storage & code gen.
+Priority customer support
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]: Is this suppose to be a list?

File: _articles/organizations/getting-started-organizations.md [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+# Set up the First and Second User Accounts
+You can choose many ways to create an account at [www.bitwarden.com](www.bitwarden.com) where you will find
+Desktop Apps for Windows, Mac, and Linux
+Browser Extensions for Google Chrome, Mozilla Firefox, Opera, Microsoft Edge, Safari, Vivaldi, Brave, Tor Browser
+Mobile Apps for iOS and Android
+Command Line Tools
+Web access through [https://vault.bitwarden.com](https://vault.bitwarden.com)
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]: another list?

File: _articles/organizations/getting-started-organizations.md [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+
+For the purpose of this tutorial, we’ll focus on the Web interface.
+## 1. Create an account for User 1
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]:
```suggestion

## 1. Create an account for User 1

```

File: _articles/organizations/getting-started-organizations.md (hunk 7 above)
Comment (go12) [reply _articles/organizations/getting-started-organizations.md]: just replicating the list of Premium features from bitwarden.com

File: _articles/organizations/getting-started-organizations.md (hunk 6 above)
Comment (go12) [reply _articles/organizations/getting-started-organizations.md]: after including it is supposed to be a list

File: _articles/organizations/getting-started-organizations.md [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+
+This tutorial walks you through the setup for two basic accounts and sharing in an Organization.
+
+## What are organizations?
+A Bitwarden organization is an entity that relates users together that want to share items. An organization could be a family, team, company, or any other type of group that desires to share items in Bitwarden.
+
+For more, please see [What are organizations?](https://help.bitwarden.com/article/what-is-an-organization/)
+
+**Are Organizations Different than a Premium Membership?**
+Yes. Organizations are for sharing items across multiple users
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]: Is this supposed to be a separate paragraph than the content below? Needs a period to end the sentence, right?

File: _articles/organizations/getting-started-organizations.md [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+Premium membership gives your individual user account additional premium features including:
+- 1GB encrypted file storage
+- Two-step login with YubiKey, FIDO U2F, & Duo
+- Password hygiene & vault health reports
+- TOTP authenticator key storage & code gen.
+- Priority customer support
+
+# Set up the First and Second User Accounts
+You can choose many ways to create an account at [www.bitwarden.com](www.bitwarden.com) where you will find
+Desktop Apps for Windows, Mac, and Linux
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]: Is this supposed to be a list? Markdown will render this differently...

File: _articles/organizations/getting-started-organizations.md [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+Browser Extensions for Google Chrome, Mozilla Firefox, Opera, Microsoft Edge, Safari, Vivaldi, Brave, Tor Browser
+Mobile Apps for iOS and Android
+Command Line Tools
+Web access through [https://vault.bitwarden.com](https://vault.bitwarden.com)
+
+For the purpose of this tutorial, we’ll focus on the Web interface.
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]: It's called the "Web Vault"

File: _articles/organizations/getting-started-organizations.md [hunk 12]
```diff
@@ -0,0 +1,196 @@
...
+  * [2. Verify Your Email](#2-verify-your-email)
+  * [3. Set up the Second User Account](#3-set-up-the-second-user-account)
+- [Setup and Share with Organizations](#setup-and-share-with-organizations)
+  * [4. Create a new Organization](#4-create-a-new-organization)
+  * [5. Add the Second User to your Organization](#5-add-the-second-user-to-your-organization)
+  * [6. Accepting the Invitation](#6-accepting-the-invitation)
+  * [7. Sharing within an Organization](#7-sharing-within-an-organization)
+- [Continuing with Bitwarden](#continuing-with-bitwarden)
+
+# Introduction
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]: We only use `##` and `###` withing the article itseld. `#` is already reserved for the large article title on the page.

File: _articles/organizations/getting-started-organizations.md [hunk 13]
```diff
@@ -0,0 +1,196 @@
...
+
+However, that second note can easily be shared with an Organization by clicking the drop down menu next to the item.
+
+{% image organizations/orgstart/7-sharing-items-with-organizations.png %}
+**Sharing Items with Organizations**
+
+# Continuing with Bitwarden
+Now that you have mastered the basics of Organizations feel free to explore all of the possibilities sharing logins and more!
+
+If you’d like to dig deeper into Bitwarden please visit [www.bitwarden.com](www.bitwarden.com).
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]: This link will not work. Need `https://`

[bitwarden.com](https://bitwarden.com)

File: _articles/organizations/getting-started-organizations.md [hunk 14]
```diff
@@ -0,0 +1,196 @@
...
+{% image organizations/orgstart/7-secure-note-user-2.png %}
+**Secure Note for User 2**
+
+Congratulations! You have now set up sharing between two users with Bitwarden Organizations!
+
+From here you can share any item using the collaborative capabilities of Bitwarden.
+
+If by chance you have an item in your personal vault that you wish to share, you can choose that option too. Here we have two items seen in the **User 1** Vault
+A secure note **GOTEST Secure Note** that has a sharing icon to the right of it indicating that is it being shared within a Collection
+A secure note **User 1 Note for Sharing** that is not part of Collection and only within the **User 1** vault
```
Comment (kspearrin) [inline _articles/organizations/getting-started-organizations.md]: Seeing this again here. Seems like you are trying to write a list or something.

**Pull request merged**